app/javascript/pages/admin/IndividualEvents/main.css

```diff
@@ -178,3 +178,20 @@ textarea {
 .muiTextField hr {
   display: none;
 }
+
+.eventsTable > div {
```

Comment:
instead of coupling to a particular structure & element, why not make a class specific for that `div` and give it a name?

Reply:
Same with the comment below, but it's because by using react tables and I wasn't able to tag elements under the api. But after a second look, seems like you can pass `ReactTable` a `style` prop which would be much cleaner. I'll see if I can do that instead.

Reply:
Okay, had another look. It seems like the api only allows you to specify styles only for a columns. I think the reason I first did that was because of the css-loader made it hard to target the actual class name ie. 'ReactTable'. I figured it out though 🎉